### ISSUE TITLE
gRPC proxy listening on external IP

### ISSUE THREAD

**Issue by patvdleer** [LN-Zap/docker-lnd#6]
gRPC proxy listening on external IP rather than anything internal, how can I specify the IP?

```
lnd-node    | 2021-03-31 13:36:58.291 [ERR] LTND: Password gRPC proxy unable to listen on 84.28.xxx.xxx:8080
lnd-node    | 2021-03-31 13:36:58.291 [ERR] LTND: unable to set up wallet password listeners: listen tcp4 84.28.xxx.xxx:8080: bind: cannot as sign requested address
lnd-node    | 2021-03-31 13:36:58.291 [INF] RPCS: Password RPC server listening on 0.0.0.0:10009
lnd-node    | 2021-03-31 13:36:58.292 [INF] BTCN: Broadcaster now active
lnd-node    | 2021-03-31 13:36:58.292 [INF] BTCN: Canceling block subscription: id=1
lnd-node    | 2021-03-31 13:36:58.324 [INF] BTCN: 6 addresses found from DNS seed testnet-seed.bluematt.me
lnd-node    | 2021-03-31 13:36:58.342 [INF] BTCN: Block manager shutting down
lnd-node    | 2021-03-31 13:36:58.342 [INF] BTCN: Address manager shutting down
lnd-node    | unable to set up wallet password listeners: listen tcp4 84.28.xxx.xxx:8080: bind: cannot assign requested address
```

```yaml
version: '3.3'
services:
  lnd-node:
    container_name: lnd-node
    image: lnzap/lnd:latest
    volumes:
      - '/etc/localtime:/etc/localtime:ro'
      - ./data:/lnd  
    command:
      - --bitcoin.active
      - --bitcoin.testnet
      - --debuglevel=info
      - --bitcoin.node=neutrino
      - --neutrino.connect=testnet1-btcd.zaphq.io
      - --neutrino.connect=testnet2-btcd.zaphq.io
      - --autopilot.active
      - --rpclisten=0.0.0.0:10009
    environment:
      - USER_ID=113
      - GROUP_ID=1001
    ports:
      - '9735:9735'
      - '10009:10009'
```

**Comment by RonaldCrb** [comment]
i think you need to add the flag `--externalip` to the command arguments, you can take a look here in the [sample config file](https://github.com/lightningnetwork/lnd/blob/master/sample-lnd.conf#L163)

**Comment by patvdleer** [comment]
thanks for the reference, fixed it with `--restlisten=0.0.0.0:8087`
